pydir/wcsfit.cpp

```diff
@@ -52,7 +76,11 @@ PYBIND11_MODULE(wcsfit, m) {
                 astrometry::SphericalICRS pole(ra, dec);
                 astrometry::Orientation orientIn(pole);
                 return astrometry::Gnomonic(orientIn.getPole(), orientIn);
-            }));
+            }))
+            .def("getOrientMatrix", [](astrometry::Gnomonic &self) {
+                const astrometry::Orientation* orient=self.getOrient();
```

Comment:
I don't think so, but do you think it would be safer to check here? For this and the next comment, they are set in the constructor and are private. Do you think there is still the danger that they could become null?